variant-caller/variant-caller-wdl-checker/topmed-variantcaller-checker.wdl

```diff
@@ -10,7 +10,9 @@ task checkerTask {
   # Cromwell error from asking for 0 disk when the input is less than 1GB
   Int additional_disk = select_first([increase_disk_size, 20])
 
-  Float disk_size = size(inputTruthVCFFile, "GB") + size(inputTestVCFFile, "GB") + additional_disk
+  Float? est_input_vcf_size
+  Float input_vcf_size = select_first([est_input_vcf_size, 30.0])
```

Comment:
@wshands I wasn't sure what the max inputTestVCFFile could be, so just inflated it to be safe.

Reply:
OK